var/spack/repos/builtin/packages/rocblas/package.py

```diff
@@ -151,6 +155,7 @@ class Rocblas(CMakePackage):
         ("@6.2.0", "dbc2062dced66e4cbee8e0591d76e0a1588a4c70"),
         ("@6.2.1", "dbc2062dced66e4cbee8e0591d76e0a1588a4c70"),
         ("@6.2.4", "81ae9537671627fe541332c0a5d953bfd6af71d6"),
+        ("@6.3.0", "df4be50f2cf7abb86b2fc7af171802a8b16e043a"),
```

Comment:
Use tensile 6.3 tag `aca95d1743c243dd0dd0c8b924608bc915ce1ae7` instead of `df4be50f2cf7abb86b2fc7af171802a8b16e043a`
https://github.com/ROCm/rocBLAS/blob/rocm-6.3.0/tensile_tag.txt